.github/workflows/application.yml

```diff
@@ -242,6 +242,7 @@ jobs:
                 php: ["8.0"]
                 symfony: ["^5.4"]
                 postgres: ["13.3"]
+                node: ["16.x"]
```

Comment:
fyi I've used `16.x` as it is LTS version and previously we didn't build assets in non-JS tests but now we have to. `16.x` and `18.x` are run in JS tests.

Reply:
What will be the difference between this one and `test-application-with-frontend`?

Reply:
@coldic3 `with-frontend` has a misleading name as it runs tests having a `@javascript` tag. Here we need assets to satisfy configuration. So the difference is significant as we run different sets of tests.

Reply:
If the only difference is with running different sets of tests maybe we can run them all in the same pipeline 🤔

Reply:
@coldic3 our Behats are already unstable, I prefer to keep them split 😂